packages/unit-vitest/package.json

```
@@ -39,18 +39,18 @@
   },
   "devDependencies": {
     "@types/lodash-es": "^4.17.12",
-    "@vitest/ui": "^2.0.5",
-    "@vue/test-utils": "^2.4.4",
+    "@vitest/ui": ">=1.6.1 <2.0.0 || >=2.1.9 <3.0.0 || >=3.0.5",
```

As you are no longer depending on `devDependencies` when installing into user projects, you can simply do what's suitable for the AE code. That constraint will resolve to the latest 3.x anyway.
```suggestion
    "@vitest/ui": "^3.0.8",
```